RTConvert/Microsoft.RTConvert.Converters/ISegmentationOutputEncoder.cs

```diff
@@ -48,11 +49,18 @@ public interface ISegmentationOutputEncoder
         /// <param name="manufacturer">The creator of the model.</param>
         /// <param name="interpreter">The interpreter of the model.</param>
         /// <returns></returns>
-        ArraySegment<byte> EncodeStructures(
-              IEnumerable<(string name, Volume3D<byte> volume, RGBColor color, bool fillHoles, ROIInterpretedType roiInterpretedType)> outputStructures,
-              IReadOnlyDictionary<string, MedicalVolume> inputChannels,
-              string modelAndVersion,
-              string institutionId,
-              string interpreter);
+        ArraySegment<byte> EncodeStructures(
+              IEnumerable<(string name, Volume3D<byte> volume, RGBColor color, bool fillHoles, ROIInterpretedType roiInterpretedType)> outputStructures,

```

Comment:
if you're messing with formatting already, maybe also have all elements of that tuple broken down by line?